crates/re_query2/Cargo.toml

```diff
@@ -0,0 +1,72 @@
+[package]
+name = "re_query2"
+authors.workspace = true
+description = "Temporary crate meant to replace re_query"
+edition.workspace = true
+homepage.workspace = true
+include.workspace = true
+license.workspace = true
+# TODO(cmc): Replace re_query with this crate. Never publish this one.
+publish = false
+readme = "README.md"
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+
+
+[features]
+default = []
+
+## Enable (de)serialization using serde.
+serde = ["dep:serde"]
+
+## Enable codegen helper binaries (generates ClampedZip & RangeZip implementations).
+codegen = ["unindent"]
+
+
+[dependencies]
+# Rerun dependencies:
+re_data_store.workspace = true
+re_format = { workspace = true, features = ["arrow"] }
+re_log_types.workspace = true
+re_types_core.workspace = true
+re_log.workspace = true
+re_tracing.workspace = true
+re_tuid.workspace = true
+
+# External dependencies:
+ahash.workspace = true
+anyhow.workspace = true
+arrow2.workspace = true
+backtrace.workspace = true
+document-features.workspace = true
+itertools = { workspace = true }
+nohash-hasher.workspace = true
+serde = { workspace = true, features = ["derive", "rc"], optional = true }
+smallvec.workspace = true
+thiserror.workspace = true
```

Comment:
Most of these aren't used yet and were just imported as-is from `re_query` -- they'll become use very soon as `re_query2` reaches feature parity.